### PR TITLE
Show location in the settings xblock.

### PR DESCRIPTION
**Pull request by oksana-slu**
@auraz

### CODE REVIEW

**Comment by auraz** [comment]
:+1: I'm not sure we should merge it to master.

**Comment by auraz** [comment]
@polesye please review

File: common/lib/xmodule/xmodule/x_module.py [hunk 1]
```diff
@@ -715,6 +717,20 @@ def get_text(value):
 
         return metadata_field_editor_info
 
+    def add_field_location(self, metadata_fields):
+        if self.system.modulestore.get_course(self.location.course_key).show_location:
+            metadata_fields['location'] = {
+                'default_value': '',
+                'explicitly_set': False,
+                'display_name': u'Location',
```
Comment (polesye) [inline common/lib/xmodule/xmodule/x_module.py]: Do we need `i18n` here?

File: common/lib/xmodule/xmodule/course_module.py [hunk 2]
```diff
@@ -740,6 +740,15 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    show_location = Boolean(
```
Comment (polesye) [inline common/lib/xmodule/xmodule/course_module.py]: Maybe `display_block_location`?

**Pull request closed without merging**